phalcon/validation.zep

```diff
@@ -124,7 +124,6 @@ class Validation extends Injectable implements ValidationInterface
 			if typeof data == "array" || typeof data == "object" {
 				let this->_data = data;
 			} else {
-				var_dump(data);
```

Comment:
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `3.0.x` branch?